sopel_remind/plugin.py

```diff
@@ -37,6 +39,48 @@ def configure(settings: Config):
         default=settings.core.homedir)
     os.makedirs(settings.remind.location, exist_ok=True)
 
+    # manage migration while configuring the plugin
+    migrate: str = input(
+        'Do you want to migrate from the built-in remind plugin? Y/n'
+    ) or 'y'
+    migrate_attr = BooleanAttribute('mock_migrate', default=True)
+    if migrate_attr.parse(migrate):
+        # is there anything to migrate?
+        builtin_filename = os.path.join(
+            settings.core.homedir,
+            settings.basename + '.reminders.db',
+        )
+
+        migrated: int = 0
+        if os.path.isfile(builtin_filename):
+            filename = backend.get_reminder_filename(settings)
+            migrated = migrate_builtin(builtin_filename, filename)
+            os.remove(builtin_filename)
+
+        if migrated:
+            print('Migrated %d reminder(s).' % migrated)
+        else:
+            print('There was no reminder to migrate. You are good to go!')
+
+
+def migrate_builtin(from_file: str, to_file: str) -> int:
+    """Migrate reminders from the built-in remind plugin."""
+    return_value: int = 0
+    reminders = backend.load_reminders(to_file)
+
+    with io.open(from_file, 'r', encoding='utf-8') as database:
+        for i, line in enumerate(database):
+            unixtime, channel, nick, message = line.split('\t', 3)
+            message = message.rstrip('\n')
+            timestamp = int(float(unixtime))  # ignore microseconds
+            reminders.append(
+                backend.Reminder(timestamp, channel, nick, message)
+            )
+            return_value = i + 1
```

Comment:
Hypothetically, if there was a way to skip a reminder that can't be imported, but still import the rest, this would be _definitely_ better. As-is, I just think it's clearer. (But I get why this code handles no errors and just aborts if one of the lines won't parse.)

```suggestion
            return_value += 1
```

Reply:
Yeah... I didn't want to bother with invalid file: it either works, or it doesn't. If it doesn't work, then the current remind plugin is already failing, and I think a human should look at the file before trying to magically define what should be or not be imported here.

Also find a better option than adding 1 to i (enumerate can start at 1...).